docs/eml-validation-refs.md

```diff
@@ -99,8 +99,8 @@ Because some of these rules cannot be enforced in XML-Schema, we have
 written a parser which checks the validity of the references and `id`s
 used in a document. This parser is included with the release of
 EML. To run the parser, you must have Java installed. To execute
-it change into the top-lvel directory of the EML release and run the
-'validate.sh' script passing your EML instance file as a parameter.
+it change into the top-level directory of the EML release and run the
+'validate.sh' script passing your EML instance file as a parameter. You can also usinvalidate your EML document from R, using the EML::validate() function.
```

Comment:
@yvanlebras Thanks for the editing. Happy to merge this in, but I'm unclear what you mean by 'usinvalidate'. Did you mean 'You can also validate your EML...'?

Reply:
Yes, sorry... My sausages-like fingers for sure  ;)

Reply:
Is it ok for you?

Reply:
That works for me. Thanks.